docs/source/issues.rst

```diff
@@ -55,7 +59,18 @@ ElasticBLAST uses cloud storage to temporally store query sequences and internal
    gsutil -m rm gs://${ELB_RESULTS_BUCKET}/logs/*  # to delete metadata files
 
 
+.. _early_shutdown:
+
 A synchronous search may shut down too early
 --------------------------------------------
 
 When doing a synchronous search (submitted with the ``--sync`` option), ElasticBLAST is continuously probing for search status to know when the search is done. When the status check times out ElasticBLAST interprets it as search failure and shuts down the cluster.
+
+.. _too_many_jobs:
+
+Too many jobs leads to a failed execution
```

Comment:
```suggestion
Too many query sequences lead to a failed execution
```
Most users will not know what a job is and how to relate it to what they are doing.

Reply:
Thanks for the suggestion but this is not accurate.